File: tests/fixtures/codegen/expected/typescript/runtime/queries/admin/events/GetEvent_test.ts
import type { GetEventTestParams, GetEventTestRow } from "../../../../types/queries/admin/events/GetEvent_test.js";
export type { GetEventTestParams, GetEventTestRow } from "../../../../types/queries/admin/events/GetEvent_test.js";
import type * as pgsid from "./pgsid/queryable.js";
export const getEventTestSql = "SELECT 2 AS value;";
export async function getEventTest(db: pgsid.Queryable): Promise<GetEventTestRow | undefined> {
    const result = await db.query(getEventTestSql, []);
    const row = result.rows[0];
    if (row === undefined)
        return undefined;
    return row as GetEventTestRow;
}

File: tests/fixtures/codegen/expected/go/queries/admin/events/geteventtest.go
package events

import "context"

const GetEventTestSQL = "SELECT 2 AS value;"

type GetEventTestParams struct {
}
type GetEventTestRow struct {
	Value int32 `db:"value"`
}

func (q *Queries) GetEventTest(ctx context.Context) (GetEventTestRow, error) {
	var row GetEventTestRow
	err := q.db.QueryRow(ctx, GetEventTestSQL).Scan(&row.Value)
	return row, err
}
